src/articles/index.ts

```diff
@@ -17,6 +17,10 @@ export class Articles extends Base {
         return this.request<Article>(`${resourceName}/${id}`)
     }
 
+    getArticleByPath (username: string, slug: string) {
+        return this.request<Article[]>(`${resourceName}/${username}/${slug}`)
```

Comment:
The type should probably be just `Article` since it returns only one entity.

```suggestion
        return this.request<Article>(`${resourceName}/${username}/${slug}`)
```